### PR TITLE
Move sync-hook to after symlink flip

### PR DESCRIPTION
**Pull request by thockin**
xref #369

### CODE REVIEW

File: .gitallowed [hunk 1]
```diff
@@ -0,0 +1 @@
+xxxyyyzzz
```
Comment (stp-ip) [inline .gitallowed:1]: Can you give context here?

Comment (stp-ip) [reply .gitallowed:1]: Why do we need this checked in?

Comment (stp-ip) [reply .gitallowed:1]: Generally would say this is more a workstation config than a repository specific one similar to editor configs, which I usually agree shouldn't be checked in.
Any reason it should be checked in besides making live easier?
(Should that be enough of a reason. Actually not sure on our stance here as this relates to easier maintainer life.)

**Comment by thockin** [comment]
I would expect anyone who uses git secrets will eventually hit this
warning?  It's not that "xxxyyyzzz" is a safe password for all repos, but
IN THIS REPO that's the example given..

**Comment by stp-ip** [comment]
Fair point.
/approve

**Pull request merged**